### DISCUSSION TITLE
"Failed to fetch" on all existing tables and any newly created tables

### DISCUSSION THREAD

**Discussion by nvenhuizen14** [AntlerVC/firetable#897] [Support / Q&A] [answered]
"Failed to fetch" on all existing tables and any newly created tables. this happened right after i ran 'RowyRun' please help me out....

edit: okay now i see that it says it was setup incorrectly

![Screenshot_20221022_113314](https://user-images.githubusercontent.com/108550117/197372181-c4e8f271-86ba-4900-8949-27127e0712c4.png)

**Comment by harinij** [comment, marked as answer]
@nvenhuizen14 were you able to setup Rowy Run?

**Reply by nvenhuizen14** [reply]
no, ive been working on n off on it for days now :/ currently i have a rowy base setup here 
![Screenshot 2022-10-25 073000](https://user-images.githubusercontent.com/108550117/197762019-49ed1d21-54dc-44b9-9b06-c1d8f8d8669c.png)



i try too to deploy it and it opens up another rowy project and window...


![Screenshot 2022-10-25 073526](https://user-images.githubusercontent.com/108550117/197763437-eaed8eb4-9a49-4818-9d21-c75f784660b2.png)


then sometimes i can get to google cloud run but now im just getting this 

![Screenshot 2022-10-25 073837](https://user-images.githubusercontent.com/108550117/197763993-344838e0-b64e-471f-ac24-9c0206967a5d.png)

even when i do get to cloud run i attempt to run and get a whole assortment of errors. :/ really bummed i was  super excited for this to workout

**Reply by harinij** [reply]
Hi @nvenhuizen14 sorry you had that experience :( We have since released a bunch of fixes and this shouldn't be the case anymore. If you are still keen you can give a try again now. If you want to set it up on the same project, you can first delete these two services from your GCP and try again: https://console.cloud.google.com/run
![Screen Shot 2023-01-12 at 11 14 27 pm](https://user-images.githubusercontent.com/307298/212064244-dcb9b2b3-e22f-41ef-9a90-d090f545ff3e.png)

**Reply by BeeBombshell** [reply]
Hey @nvenhuizen14, I found a similar discussion here: https://github.com/rowyio/rowy/discussions/1087. Could you try out the steps as suggested? Hope this helps!
